testing/armagetronad/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: Díaz Urbaneja Diego <sodomon2@gmail.com>
+pkgname=armagetronad
+pkgver=0.2.8.3.4
+pkgrel=0
+pkgdesc="A Tron Clone in 3D."
+url="http://armagetronad.net/"
+arch="x86 x86_64"
```

Comment:
Why doesn't it work on other arches?

Reply:
doesn't work in other arcs, no enrialidad if they work what happens is that I don't do the packages in a chroot and I have the architecture x86_64

Reply:
CI can test on other arches.